mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -89,7 +87,7 @@ namespace mbf_abstract_nav
      * @param tf_listener_ptr Shared pointer to a common tf listener
      */
     AbstractControllerExecution(
-        const std::string name,
```

Comment:
format of `const std::string& name`

Note that parameter 'name' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions clang-tidy

Reply:
Seems to be different throughout files, do you use a clang-format or something?